sources/core/objects/Floor.cpp

```diff
@@ -77,7 +79,6 @@ void Floor::SetOpen(bool o)
     //qDebug() << "End setopen";
 }
 
-
 Plating::Plating(quint32 id) : Floor(id)
 {
     open_ = true;
```

Comment:
Proper `floor_type_` should be set for `Plating` too - because otherwise `Floor::AfterWorldCreation()` will set `Plating` state as `"floor"` - and they look same in the game.